### ISSUE TITLE
New Program Card 

### ISSUE THREAD

**Issue by divppc** [divppc/Physio-App#72]
https://app.zeplin.io/project/5911792e3ad67ec053171df6/screen/59ca0939aa08dcb3ea07ec47

**Comment by divppc** [comment]
missing icons http://joxi.ru/E2pVZgzh9ynB5r

**Comment by divppc** [comment]
also this http://joxi.ru/n2YnJ5efoBLdyA

**Comment by sonit1810** [comment]
Please recheck, he just updated

**Comment by divppc** [comment]
Hi Sonit, here are should be two different lists, right? http://joxi.ru/823bYp9fJlNvom
one for exercises and second for notes, or how you need it

**Comment by sonit1810** [comment]
I think they are just items, in same area div. Because i think Pain Scale
just an option of program, notes is the same too, but exercises belong to
program too, that's why they are in same area. Please ask Justin if you
don't understand, i don't understand too because i did not design that
screen, Justin worked with design, he gave design file, then later i must
analysis myself what is it and how can i do it...

**Comment by divppc** [comment]
understood ok, i'll ask him, thanks

**Comment by sonit1810** [comment]
okay Ihor, thanks, but i think it works like i said :D

**Comment by divppc** [comment]
Hello Sonit, i'm almost finish new card page. Want to ask you about js functions on this page, will you do it or i should? Thanks

**Comment by divppc** [comment]
i already added collapsing, tabs, range slider, and custom input type number
http://physio.div.agency/card.html

**Comment by sonit1810** [comment]
Hi Ihor,

What are js functions you meant? Example drag & drop?

**Comment by divppc** [comment]
yes, also modal window, remove item from list, calendar date picker - everything which was created in previous program builder template

**Comment by sonit1810** [comment]
Modal windows i can do. 

Calendar you should do. 

What was not exised in previous program builder page i can do , else please do it

Thanks.

**Comment by divppc** [comment]
ok

**Comment by divppc** [comment]
done http://physio.div.agency/card.html

**Comment by sonit1810** [comment]
Hi Ihor,

Exercise type & part should be buttons as before, in new design part is select list but actually it is button, stupid designer, please fix this, i think we can put old style/elements to it ?

![image](https://user-images.githubusercontent.com/8002065/31053725-33c97ff2-a6ce-11e7-8593-9dbbcc6cf0a7.png)

**Comment by sonit1810** [comment]
In old layout, "Time per week" actually is select list, because there are 3 options for it, please make the same in new layout

![image](https://user-images.githubusercontent.com/8002065/31053735-575f9fdc-a6ce-11e7-823f-eb7d78c2677b.png)

![image](https://user-images.githubusercontent.com/8002065/31053743-664e7e8c-a6ce-11e7-9900-71e830594985.png)

@divppc  can you fix this ?

**Comment by sonit1810** [comment]
We need add another slider in the box too

![image](https://user-images.githubusercontent.com/8002065/31053765-f2950cf8-a6ce-11e7-9271-7507faf14fc5.png)

in old version we have maximum 2 slider 
- Rep based : Reps & Time
- Time based : Hold time

![image](https://user-images.githubusercontent.com/8002065/31053775-2589b398-a6cf-11e7-8b9f-512d64a011f3.png)

**Comment by divppc** [comment]
Hi Sonit. I did what you ask http://physio.div.agency/card.html

**Comment by sonit1810** [comment]
k thank you!

**Comment by sonit1810** [comment]
Hi Ihor,

We need plus icon (+) in  list tool (like it had before, for exercise list and article list) when hover, so we can add that item into program

![image](https://user-images.githubusercontent.com/8002065/31086252-8b08e03c-a7c3-11e7-8fec-7607e80b3f83.png)

**Comment by divppc** [comment]
done http://physio.div.agency/card.html#

**Comment by sonit1810** [comment]
Hi Ihor,
before edit mode show only edit pencil
when in edit mode change edit pencil to disk (for save) and other circle show reload/refresh style arrow for cancel.
![image](https://user-images.githubusercontent.com/8002065/31214188-58cfa0ca-a9d4-11e7-8be5-c4a0df41489a.png)

**Comment by divppc** [comment]
Hi Sonit.  First time, user see only one pencil and program settings don't available for changes. If pencil is clicked then we see two icons, save and refresh, right? What about remove icon?

**Comment by sonit1810** [comment]
Yes. Leave remove icon Ihor. Actually Justin ask for popup to show icons and text bellow icons, can you try?

**Comment by divppc** [comment]
added edit function, try here http://physio.div.agency/card.html
but i don't understand about popup, what you mean?

**Comment by sonit1810** [comment]
You can make a popup with that icons, and text below icon, i will handle
it. It popup when editing exercise but does not save, and select another
exercise.

We need active class for item in the list too, it was green border when
selected before.

**Comment by divppc** [comment]
still don't understand about what popup you talking about.., so i need design for it, please ask designer.

**Comment by sonit1810** [comment]
Forget it, it's Justin idea. When you editing exercise, but you don't save, then you click to another exercise in list, it will popup to ask. In popup window, you have 2 icon & text. Icon in 1 row, text in 1 row, and text bellow icon. 1 icon for save, 1 icon ok no save...idea like that. I don't know, that's Justin idea. So you can ignore it now. No need do it, we can do later.

when in edit mode change edit pencil to disk (for save) and other circle show reload/refresh style arrow for cancel. <-- we need ICON FOR RELOAD too, i saw you show save icon when click edit icon, missing reload icon, just put it near save icon, i will do js to show/hide it later.

**Comment by divppc** [comment]
ok, sorry, i forgot reload icon

**Comment by divppc** [comment]
added reload icon

**Comment by sonit1810** [comment]
Hi Ihor,

It seems you removed delete icon with latest changes ? Can you add delete icon back ?

![image](https://user-images.githubusercontent.com/8002065/31330927-8c929e24-ad0a-11e7-9ce6-43220e41f781.png)

**Comment by sonit1810** [comment]
We need make "Time per Week" as drop down list too, when click to Text, like it does before

![image](https://user-images.githubusercontent.com/8002065/31331248-cfb0646a-ad0b-11e7-90af-18326c1a23a3.png)

**Comment by divppc** [comment]
Hi Sonit. Fixed, sorry

**Comment by sonit1810** [comment]
Hi Ihor,

You made wrong action again. Don't split Time per Week like latest changes, it's correct at previous, just make text "Times per week" if select list, it has 3 options before. The value Time, For, Start date, already there in new layout, please see image below, you just need make Time per Week as select list, and style it so it look as text (remove border...) ?

![image](https://user-images.githubusercontent.com/8002065/31334435-873d7ed6-ad18-11e7-965d-ac9e44d75a85.png)

This is wrong, don't split it to Frequency and  Times, keep as new design

![image](https://user-images.githubusercontent.com/8002065/31334509-c58811c4-ad18-11e7-8805-a19abd0a01e0.png)

**Comment by divppc** [comment]
like this? http://joxi.ru/Grqzag1SQE9agm

**Comment by sonit1810** [comment]
No Ihor, what's wrong with you ? You made it before, now you still don't understand ?

Keep it as new layout, as image bellow (you made this already)

![image](https://user-images.githubusercontent.com/8002065/31344489-0dd6dfe0-ad3d-11e7-9f47-ae8774bbf84c.png)

And, "Time per week" actually is another drop down list, normal it look like this

![image](https://user-images.githubusercontent.com/8002065/31344535-2ea05274-ad3d-11e7-9bb8-c77315807624.png)

When click to it it will drop down

![image](https://user-images.githubusercontent.com/8002065/31344628-6b5dabe4-ad3d-11e7-9d23-96e1a6a3be73.png)

Make sense ? We need in new layout, "Time per week" is a drop down too, normal look like text, and click is drop down (you styled it by css, remove border of select list....)

**Comment by divppc** [comment]
Perhaps the problem is that you do not explain well? Have you thought about this?

**Comment by sonit1810** [comment]
Actually it's design problem. Justin worked with design, and made as him
want, and i must analysis and if anything not correct i will tell him and
ask you correct it.

So, do you understand now ?

**Comment by sonit1810** [comment]
I write in image so i hope you can understand

![image](https://user-images.githubusercontent.com/8002065/31346095-2903a32a-ad42-11e7-8b4c-a5a7d2911a53.png)

**Comment by divppc** [comment]
attempt №3 http://physio.div.agency/card.html

**Comment by sonit1810** [comment]
Yes, but correct 50% Ihor,

Please keep it position, don't move it to same row with other drop down list, keep its position as design, but make it as drop down list (as you already did).

![image](https://user-images.githubusercontent.com/8002065/31367086-1ca727de-ad9f-11e7-80c5-b0f6c493c258.png)

**Comment by divppc** [comment]
done

**Comment by sonit1810** [comment]
Thanks Ihor, it works.

But in future please don't use css for fixed id, because i will change id of element then css will not affected, thanks.

![image](https://user-images.githubusercontent.com/8002065/31429153-66445c1a-ae97-11e7-9fa9-f1b1dc0ef38e.png)
